xnat_tools/dicom_export.py

```diff
@@ -164,6 +168,11 @@ def dicom_export(
     if validate_frames:
         validate_frame_counts(scans, export_session_dir)
 
+    # If a configuration file is passed, correct DICOM headers of
+    # specified files
+    if correct_dicoms_config != "":
```

Comment:
Minor suggestion.  In Python, the empty string is _falsy_, meaning this conditional statement can be simplified to:

```
if correct_dicoms_config:
```